daemon/emer-daemon.c

```diff
@@ -1432,31 +1361,13 @@ emer_daemon_class_init (EmerDaemonClass *klass)
                          G_PARAM_STATIC_STRINGS);
 
   /* Blurb string is good enough default documentation for this */
-  emer_daemon_props[PROP_SINGULAR_BUFFER_LENGTH] =
-    g_param_spec_int ("singular-buffer-length", "Buffer length singular",
-                       "The number of events allowed to be stored in the "
-                       "individual metric buffer",
-                       -1, G_MAXINT, DEFAULT_METRICS_PER_BUFFER_MAX,
-                       G_PARAM_CONSTRUCT_ONLY | G_PARAM_WRITABLE |
-                       G_PARAM_STATIC_STRINGS);
-
-  /* Blurb string is good enough default documentation for this */
-  emer_daemon_props[PROP_AGGREGATE_BUFFER_LENGTH] =
-    g_param_spec_int ("aggregate-buffer-length", "Buffer length aggregate",
-                      "The number of events allowed to be stored in the "
-                      "aggregate metric buffer",
-                      -1, G_MAXINT, DEFAULT_METRICS_PER_BUFFER_MAX,
-                      G_PARAM_CONSTRUCT_ONLY | G_PARAM_WRITABLE |
-                      G_PARAM_STATIC_STRINGS);
-
-  /* Blurb string is good enough default documentation for this */
-  emer_daemon_props[PROP_SEQUENCE_BUFFER_LENGTH] =
-    g_param_spec_int ("sequence-buffer-length", "Buffer length sequence",
-                      "The number of events allowed to be stored in the "
-                      "sequence metric buffer",
-                      -1, G_MAXINT, DEFAULT_METRICS_PER_BUFFER_MAX,
-                      G_PARAM_CONSTRUCT_ONLY | G_PARAM_WRITABLE |
-                      G_PARAM_STATIC_STRINGS);
+  emer_daemon_props[PROP_MAX_BYTES_BUFFERED] =
+    g_param_spec_ulong ("max-bytes-buffered", "Max bytes buffered",
+                        "The maximum number of bytes of event data that may be "
+                        "buffered in memory. Does not include overhead.",
+                        0ul, G_MAXULONG, 100000ul /* 100 kB */,
```

Comment:
It's the same value, but it is just a coincidence that we agreed the metrics system can use 100 kB of RAM and upload at most 100 kB at a time.